### ISSUE TITLE
Warning message for text-hide() mixin thet has been deprecated

### ISSUE THREAD

**Issue by PaoloPgn** [understrap/understrap#642]
Hi
I got this error running gulp watch
"WARNING: The `text-hide()` mixin has been deprecated as of v4.1.0. It will be removed entirely in v5"
I googled and I find that is a Bootstrap related issue that can be solved within _text-hide.scss changing line 10, but someone don't like it cause they say it's better to use --quiet mode.
Is this possible with understrap and how?

## Issue is about..
  - [ ] a Bug Report

  - WordPress Version: 4.9.x
  - OS Version: MacOsX 10.11.6
  - Language:  S/CSS 

## *What* is affected by this bug?
Bootstrap

## *When* does this occur?
when you run gulp watch

## *Where* on the platform does it happen?
inseide gulp messages
## *How* do we replicate the issue?
I don't know, try on a fresh install
# Todos
change line 10 in _text-hide or run bootstrap in quite mode

**Comment by holger1411** [comment]
Hi!
We will do exactly what the Bootstrap guys do: Keeping it but with a deprecate message, than replace it with 5.0. The message just gives the user an important info: "Watch out, this component will be removed in the future! Don´t rely to much on it."

So adding a workaround (which adds inconsistency to "our" Bootstrap version and make it un-updateable from the original) just to get rid of a important message isn´t a proper way.

And there is already a PR for Bootstrap 4 here:
https://github.com/twbs/bootstrap/pull/26242
Maybe we don´t have to wait till 5.0

But if you want to do it on your own you can follow the solution you pointed out above.
Open https://github.com/understrap/understrap/blob/master/src/sass/bootstrap4/mixins/_text-hide.scss
Do the changes and run
`gulp styles`
to rebuild your css

**Comment by holger1411** [comment]
@PaoloPgn Yes I know 😄  Was the same here. As @mdo announced it was the first time they use a deprecate message.

**Comment by axlright** [comment]
The deprecation message is written directly into _text-hide.scss - I'm sure there are a *lot* of Bootstrap devs out there that are sick of looking at that message.

I got tired of looking at it so I commented out lines 56-58 of /src/sass/bootstrap4/utilities/_text.scss

**Comment by Thomas-A-Reinert** [comment]
Guess I seen it a zillion times today :)

Anyway, I guess that´s pretty good practise. Maybe we could learn from that!?
